Library/Formula/libxmp-lite.rb

```diff
@@ -0,0 +1,10 @@
+class LibxmpLite < Formula
```

Comment:
Unfortunately, for technical reasons we can't support formula renames. Mind doing this the other way around? E.g. add an alias under the `libxmp-lite` name, and leave the formula itself as `coremod`.